### ISSUE TITLE
//naturalize only does exposed surfaces to the sky.  I require it to work inside caves as well.

### ISSUE THREAD

**Issue by nikitis** [EngineHub/WorldEdit#1711]
### The Problem

I have a cavernous map and I wish to have the ground level naturallized.  I have it within my cuboid selection but when I go to //naturalize it only will cover areas that are exposed to the sky which is like only 5% of my map.  And it leaves patchy blotches which look bad.  

As It was explained to me this effect is occurring because World Edit starts from the highest Y and works downward.  But it won't go below the first one it encounters.   I need this effect to work even if there is exposed stone underneath more stone like the ground floor of a cave.  This can make caves look far more lush.  

### A Solution

Redesign the logic so that it will naturalize any stone that has 1 air block above it and only above it (not sides or underside), and is within your cuboid selection. This way it will work for caves and give caves a grassy / dirt layers.  

Maybe add it as an option with -all    so    //naturalize -all so it will not use logic to only do stone exposed to the sky downwards but also stone and the 3 blocks below it with dirt even in caves if the cave stone is selected.

**Comment by octylFractal** [comment]
> only will cover areas that are exposed to the sky
This is false, as I stated multiple times on Discord. It goes from the top of your selection, you are free to move your selection to cover underground areas.
